franka_control/src/franka_control_node.cpp

```diff
@@ -42,9 +43,16 @@ int main(int argc, char** argv) {
   std::copy(joint_names_vector.cbegin(), joint_names_vector.cend(), joint_names.begin());
 
   std::string robot_ip;
-  node_handle.getParam("robot_ip", robot_ip);
+  if (!node_handle.getParam("robot_ip", robot_ip)) {
+    ROS_ERROR("Invalid or no robot_ip parameter provided");
+    return 1;
+  }
+
   std::string arm_id;
-  node_handle.getParam("arm_id", arm_id);
+  if (!node_handle.getParam("arm_id", arm_id)) {
```

Comment:
Isn't this unnecessary now?

Reply:
Internally, we still use the arm id to name some ros_control handles, it's just not used for the ROS namespaces anymore. To avoid these larger code changes for now, I only removed it from the launchfiles and added defaults to the rosparam configs instead.